File: 3st/Juram/programmer/배열 회전시키기.js
function solution(numbers, direction) {
    var answer = [];
    if(direction == "right"){
        for(let i=1; i<numbers.length; i++){
            answer[0] = numbers[numbers.length-1];
            answer.push(numbers[i-1]);
        }
    }
    else if(direction == "left"){
        for(let i=1; i<numbers.length; i++){
            answer.push(numbers[i]);
        }
        answer.push(numbers[0]);
        
    }
    return answer;
}

File: 3st/Hyeonseo/programmer/배열 회전시키기.py
def solution(numbers, direction):
    if direction == "right":
        return [numbers[-1]] + numbers[:-1]  # 마지막 원소 앞으로
    else:  # direction == "left"
        return numbers[1:] + [numbers[0]]    # 첫 원소 뒤로
